ipset.go

```diff
@@ -287,6 +299,7 @@ func (s *IPSetBuilder) IPSet() (*IPSet, error) {
 	if len(s.errs) == 0 {
 		return ret, nil
 	} else {
+		s.errs = s.errs[:0:0]
```

Comment:
This seems wrong. You're clearing it before returning it?

Reply:
Oops. Fixed, thanks. The test that would have caught this bug is in #188.